packages/reports/addon/components/report-actions/multiple-format-export.ts

```diff
@@ -157,12 +163,14 @@ export default class MultipleFormatExport extends ReportActionExport {
   @task *gSheetExportTask(): TaskGenerator<void> {
     const { naviNotifications } = this;
 
-    const response = yield fetch(this.gsheetExportHref);
-    const json = yield response.json();
+    console.log('gsheetExportHref');
+    const response = yield taskFor(this.gsheetExport.fetchAndPollGsheet).perform(
+      new URL(this.gsheetExportHref, window.location.origin)
+    );
 
     naviNotifications?.clear();
     naviNotifications?.add({
-      title: json.url ? `Your export is done and available at ${json.url}` : 'Your export has finished!',
+      title: response.url ? `Your export is done and available at ${response.url}` : 'Your export has finished!',
```

Comment:
Update to https://github.com/denali-design/denali-ember/releases/tag/v1.0.0-alpha.26

```suggestion
      title: 'Your export has finished!',
      context: response.url ? `Click <a href="${response.url}" target="_blank" rel="noopener noreferrer">here to view it &raquo;</a>` : undefined
```